pkg/packaging/target_test.go

```diff
@@ -143,15 +143,23 @@ func TestTargetParse(t *testing.T) {
 	}{
 		{
 			in:  "darwin-launchd-pkg",
-			out: &Target{Platform: Darwin, Init: LaunchD, Package: Pkg},
+			out: &Target{Platform: Darwin, Init: LaunchD, Package: Pkg, Arch: Universal},
 		},
 		{
 			in:  "windows-none-msi",
 			out: &Target{Platform: Windows, Init: NoInit, Package: Msi},
 		},
 		{
 			in:  "linux-systemd-rpm",
-			out: &Target{Platform: Linux, Init: Systemd, Package: Rpm},
+			out: &Target{Platform: Linux, Init: Systemd, Package: Rpm, Arch: Amd64},
+		},
+		{
+			in:  "linux-arm64-systemd-rpm",
+			out: &Target{Platform: Linux, Init: Systemd, Package: Rpm, Arch: Arm64},
+		},
+		{
+			in:         "linux-notavalidarch-systemd-deb",
```

Comment:
I read this like a proper name "Notava Lidarch" and thought... what the hell kind of obscure linux distro / install format are we trying to support now?